src/Orchard.Web/Modules/Orchard.Demo/orchard.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "$schema": "orchard-schema.json",
+
+  "minOrchardVersion": "2.0.0",
+  "module": {
+    "antiForgery": false,
+    "path": "OrchardDemo",
+    "features": [
+      {
+        "id": "Orchard.Demo",
+        "name": "Demo",
+        "description": "Demo description",
+        "category": "Demo",
+        "priority": 0,
+        "dependencies": [
+          "Orchard.Demo.Commands"
+        ]
+      }
+    ]
+  }
+}
```

Comment:
There is something about this file that doesnt read right.... I think it might be 'path' but... hmm... so this means we are decoupling the name of the folder and the name of the project.... so I could have FooModule/BarProject.xproj for example? where as before it needed to be BarProject/BarProject.xproj......